backend/engine/plugins/gitlab_repo_health/rules/first_order_rules/branch_protection_prevent_secret_files.py

```diff
@@ -0,0 +1,38 @@
+from requests import HTTPError
```

Comment:
Nit: This rule doesn't seem to be documented in the Readme with the other rules

Reply:
Added to README